packages/manager/src/features/ObjectStorage/SummaryLanding/SummaryLanding.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Typography, useTheme } from '@linode/ui';
+import React from 'react';
+
+import { Link } from 'src/components/Link';
+
+import { EndpointMultiselect } from './Partials/EndpointMultiselect';
+import { EndpointSummaryRow } from './Partials/EndpointSummaryRow';
+
+import type { EndpointMultiselectValue } from './Partials/EndpointMultiselect';
+
+export const SummaryLanding = () => {
+  const theme = useTheme();
+
+  const [selectedEndpoints, setSelectedEndpoints] = React.useState<
+    EndpointMultiselectValue[]
+  >([]);
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: theme.bg.bgPaper,
+        padding: theme.spacingFunction(24),
+        display: 'flex',
+        flexDirection: 'column',
+        gap: theme.spacingFunction(16),
+      }}
```

Comment:
optional: remove L12 and the `useTheme` import in favor of this:


```suggestion
      sx={(theme) => ({
        backgroundColor: theme.bg.bgPaper,
        padding: theme.spacingFunction(24),
        display: 'flex',
        flexDirection: 'column',
        gap: theme.spacingFunction(16),
      })}
```

Reply:
Good idea, thank you.